packages/calendar/test/todolist-1.spec.js

```diff
@@ -0,0 +1,221 @@
+function getPossibleExpressions(expressionType, count) {
+    const data = require('../data/expressions/en.json')
+    switch (expressionType) 
+    { 
+        case "create list":
+            return data.todolist.create_list.expressions
+        case "view lists":
+            return data.todolist.view_lists.expressions
+        case "view list":
+            return data.todolist.view_list.expressions
+        case "rename list":
+            return data.todolist.view_list.expressions
+        case "delete list":
+            return data.todolist.delete_list.expressions
+        case "add todos":
+            return data.todolist.add_todos.expressions
+        case "complete todos":
+            return data.todolist.complete_todos.expressions
+        case "uncheck todos":
+            return data.todolist.uncheck_todos.expressions
+        default:
+            console.log("invalid expression type requested")
+            break
+    }
+}
+
+function getOutputs(){
+    const fs = require('fs')
+    const path = require('path')
+    let data = JSON.parse(fs.readFileSync(path.resolve(__dirname, '../data/db/calendar.spec.json')))
+    return data
+}
+
+async function testCreateListExpressions(count) {
+    const expression = getPossibleExpressions("create list")[count]
+    var quote = '\"'
+    var expectedIndex = quote.concat(count.toString()).concat('\" list')
+    global.nlu.brain.execute = jest.fn()
+    await global.nlu.process(expression.replace("x", count))
+    var [obj] = global.nlu.brain.execute.mock.calls
+    await global.brain.execute(obj[0])
+    return [global.brain.finalOutput, expectedIndex]
+}
+
+async function testViewListsExpressions(count) {
+    const expression = getPossibleExpressions("view lists")[count]
+    var quote = '\"'
+    global.nlu.brain.execute = jest.fn()
+    await global.nlu.process(expression)
+    var [obj] = global.nlu.brain.execute.mock.calls
+    await global.brain.execute(obj[0])
+    return global.brain.finalOutput
+}
+
+async function testExpression(expression) {
+    global.nlu.brain.execute = jest.fn()
+    await global.nlu.process(expression)
+    var [obj] = global.nlu.brain.execute.mock.calls
+    await global.brain.execute(obj[0])
+}
+
+async function testExpressions(expressions) {
+    for (let i = 0; i < expressions.length; i++){
+        await testExpression(expressions[i])
+    }
+}
+
+function createObjs(){
```

Comment:
Prefer explicit than implicit, so instead you could call this function `createObjects`.
Also it doesn't seems like this function "only" create objects, it actually creates todos/lists objects, so be more explicit of what the function does in its name.